### ISSUE TITLE
Error running skycoin/skycoin:arm32v6 image on BeagleBoneBlack

### ISSUE THREAD

**Issue by olemis** [skycoin/skycoin#1527]
Steps to reproduce issue

```
$ sudo docker pull skycoin/skycoin:develop-arm32v6
develop-arm32v6: Pulling from skycoin/skycoin
95d54dd4bdad: Pull complete
72bf7d76c392: Pull complete
0c106b780ae2: Pull complete
b66824423c78: Pull complete
a72063fef82d: Pull complete
$ sudo docker run -ti --rm -p 6000:6000 -p 6420:6420 skycoin/skycoin:develop-arm32v6 /bin/bash
su: can't execute 'in/nologin': No such file or directory
```

**Comment by olemis** [comment]
@stdevMauricio pls provide us with further details

**Comment by olemis** [comment]
@stdevMauricio any progress on fixing this ?

**Comment by olemis** [comment]
@gz-c this issue was already solved after merging #1538
